FrontDesk/src/FrontDesk.Api/OfficeSettings.cs

```diff
@@ -6,6 +6,6 @@ namespace FrontDesk.Api
     public class OfficeSettings : IApplicationSettings
     {
         public int ClinicId { get { return 1; } }
-        public DateTime TestDate { get { return new DateTime(2014, 6, 9); } }
+        public DateTime TestDate { get { return new DateTime(2030, 9, 23); } }
```

Comment:
date changed in api and seed but you need to change the date manually in the appointments page to see the data or we need to create api to get the test date and use there?

Reply:
Change it manually from what? Does it always default to today? Have it start out on that date, every time.

Reply:
I changed to force blazor to read from api what is today's date. in PR #5